project/src/main/webapp/event-card.html

```diff
@@ -84,13 +105,12 @@ <h5 class="card-title" id="event-card-title">
         </p>
         <div class="display-inline-block" id="event-card-labels">
         </div>
+        <br />
         <ul class="list-group list-group-flush" id="details">
-          <button class="btn">
-            <a href="event-details.html" id="event-details">See more details</a>
-          </button>
-          <button type="button" class="btn btn-primary">
-            <a href="event-details.html" id="event-register">Register</a>
-          </button>
+          <div class="btn-group" role="group">
+            <a class="btn btn-outline-primary" href="event-details.html" id="event-details">Details</a>
```

Comment:
just want to confirm that these links are still set in a js file as something like 'event-details.html?eventId= XXX' and not just plain 'event-details.html'

Reply:
Good catch - yup, this PR keeps those links now